people/MarcoIeni.toml

```diff
@@ -1,5 +1,5 @@
 name = "Marco Ieni"
-github = "MarcoIeni"
+github = "marcoieni"
```

Comment:
The github username should ideally match the name of the file.

Reply:
Good point, thanks!